libqtile/window.py

```diff
@@ -101,6 +109,7 @@ def get_attr(self):
             # trigger the geometry setter on all these
             self.x = g.x
             self.y = g.y
+            self.z = g.z
```

Comment:
How is this supposed to work?

Reply:
I guess you mean that `get_geometry()` wouldn't return a z-property? I hadn't noticed that before, should definitely use the default then.

Reply:
Yes, that's what I meant.

Reply:
As discussed in the other comment, I think we can pretend _z will never be None and just drop the addition.